services/rest/src/main/java/mil/nga/giat/geowave/service/rest/FileUpload.java

```diff
@@ -106,6 +106,11 @@ public UploadedFile accept(
 			// 3/ Request is parsed by the handler which generates a
 			// list of FileItems
 			String tempDir = System.getProperty("java.io.tmpdir");
+			// Linux returns temp dir as '/tmp' but need to add '/' after to
+			// separate file name
+			if (tempDir.charAt(tempDir.length() - 1) != '/' && tempDir.charAt(tempDir.length() - 1) != '\\') {
+				tempDir = tempDir + '/';
+			}
```

Comment:
Haven't tested this yet, but maybe we could change line 120 to: 
result = new UploadedFile(filename.getAbsolutePath());